src/Products/Signature/Signer/ImageSigner.cs

```diff
@@ -75,19 +64,16 @@ public override SignOptions SignCells()
         /// <returns>SignOptions</returns>
         public override SignOptions SignSlides()
         {
-            // setup image signature options with relative path - image file stores in config.ImagesPath folder
-            SlidesSignImageOptions signOptions = new SlidesSignImageOptions(signatureData.SignatureGuid);
-            SetOptions(signOptions);            
-            return signOptions;
+            return SignWord();
         }
 
-        private void SetOptions(SignImageOptions signOptions)
+        private void SetOptions(ImageSignOptions signOptions)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'SetOptions' a static method.](https://app.codacy.com/gh/groupdocs-signature/GroupDocs.Signature-for-.NET-MVC/pullRequest?prid=5325146)